core/web/presenters/job.go

```diff
@@ -54,6 +55,7 @@ type FluxMonitorSpec struct {
 	PollTimerDisabled bool                `json:"pollTimerDisabled"`
 	IdleTimerPeriod   string              `json:"idleTimerPeriod"`
 	IdleTimerDisabled bool                `json:"idleTimerDisabled"`
+	MinPayment        *assets.Link        `json:"minPayment"`
```

Comment:
Could we avoid a pointer here? 0 is the same as `nil` for this I believe.

Reply:
`minPayment` is an optional field when creating the job, and is stored as a `NULL` in the DB when not present. On the front end side, passing a `null` down allows the TOML generator to omit that field in the Job Definition tab.

We could make this only accept a value, but would have to change how we handle the creation, which is out of the scope of this PR. I'm also not sure how the TOML parser handles unmarshalling empty fields to values (I assume it would just put the zero value in but would have to check)

I'm good to make the change separately if you think its a better implementation

Reply:
One reason to keep it a pointer is just that the json marshalling methods are defined as pointer receivers (and used as such in a bunch of other models), so it'd be a bigger change to make it just a struct. I'm ok just leaving it as a pointer